Expensify/App
# VBA Company Step (UI only)

### Details
Implements the UI portion of the VBA Company Step.

### Fixed Issues
Fixes (partial) https://github.com/Expensify/Expensify/issues/166916

### Tests / QA Steps
1. Go to `$expensifyCashURL/bank-account/`
1. Click on the bank icon
1. In the plaid modal, click "Chase"
1. Enter `user_good` and `pass_good` as login credentials.
1. Look at the shiny `Company Information` form on the next page.
1. Verify that each form element can be interacted with, and that the submit button or active form input is never covered by the software keyboard. 

### Tested On


### Screenshots
#### Web
![image](https://user-images.githubusercontent.com/47436092/122280327-ef01ec00-ce9d-11eb-9a9b-56564968d377.png)

![image](https://user-images.githubusercontent.com/47436092/122280376-fb864480-ce9d-11eb-96dd-17d435110fba.png)


#### Mobile Web


#### Desktop


#### iOS


#### Android
